Makefile

```diff
@@ -365,6 +370,9 @@ $(MUSL_PRINTSCAN_NO_FLOATING_POINT_OBJS): CFLAGS += \
 	    -D__wasilibc_printscan_no_floating_point \
 	    -D__wasilibc_printscan_floating_point_support_option="\"remove -lc-printscan-no-floating-point from the link command\""
 
+$(BULK_MEMORY_OBJS): CFLAGS += \
+        -mbulk-memory
```

Comment:
I think if safe/better to pass this to all object files.   No reason not to do that is there?

Reply:
I've encountered `fatal error: error in backend: only -ftls-model=local-exec is supported for now on non-Emscripten OSes: variable errno` when compiling some sources with `-mbulk-memory`

Reply:
That sounds like its worth looking into.   I don't see why adding `-mbulk-memory` would generate that error.  Would you mind opening an llvm issue about that?

Reply:
Sure, but LLVM bugzilla seems to prohibit new logins for now. The CI log with the clang-13 panicking command line can be found at https://github.com/tweag/wasi-libc/runs/4420917003?check_suite_focus=true#step:8:8. In the meantime, I believe it makes sense to resolve this thread and keep the current workaround as it is?

Reply:
I guess I don't mind landing this patch as workaround for this clang issue, but I would like to be able to add a TODO to the Makefile along with a link to a bug.  

Perhaps a TODO with an explanatory comment would be enough?  Something like: "Limit the use of this feature to just a few compilation units due to clang crash when we try to use it throughout.  TODO: Remove this once clang gets fixed"

Is there some reason you need this patch to land sooner rather than later?

Reply:
Fair enough. I have no particular reason to rush getting this merged since we're using our fork anyway, this PR is just meant to avoid duplicate upstream work. I'm fine with waiting for LLVM bug to be filed and fixed then applying `-mbulk-memory` globally.